### PR TITLE
LibWeb/CSS: Treat counters() function with 1 argument as invalid

### LINKED ISSUE #887: CSS: Crash when `counters` function is provided only one argument
CC @AtkinsSJ 

Repro:
```html
<style>
    html {
        content: counters(index);
    }
</style>
```

```
VERIFICATION FAILED: i < m_size at ladybird/AK/Vector.h:148
liblagom-ak.so.0(ak_verification_failed+0xef) [0x72846558344f]
liblagom-web.so.0 Web::CSS::Parser::Parser::parse_counter_value(Web::CSS::Parser::TokenStream<Web::CSS::Parser::ComponentValue>&) 0xc2d) [0x7284665be64d]
liblagom-web.so.0 Web::CSS::Parser::Parser::parse_css_value_for_properties(AK::Span<Web::CSS::PropertyID const>, Web::CSS::Parser::TokenStream<Web::CSS::Parser::ComponentValue>&) 0x33e) [0x7284665c3e9e]
liblagom-web.so.0 Web::CSS::Parser::Parser::parse_css_value_for_property(Web::CSS::PropertyID, Web::CSS::Parser::TokenStream<Web::CSS::Parser::ComponentValue>&) 0x31) [0x7284665c17e1]
liblagom-web.so.0 Web::CSS::Parser::Parser::parse_content_value(Web::CSS::Parser::TokenStream<Web::CSS::Parser::ComponentValue>&) 0x1d5) [0x7284665c8f75]
```

### PR DESCRIPTION
**Pull request by AtkinsSJ**
Fixes #887.

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp [hunk 1]
```diff
@@ -3003,7 +3003,7 @@ RefPtr<StyleValue> Parser::parse_counter_value(TokenStream<ComponentValue>& toke
         auto& function = token.function();
         TokenStream function_tokens { function.values() };
         auto function_values = parse_a_comma_separated_list_of_component_values(function_tokens);
-        if (function_values.is_empty() || function_values.size() > 3)
+        if (function_values.is_empty() || function_values.size() < 2 || function_values.size() > 3)
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp:3006]: Minor thing, but this makes the `is_empty` check redundant
```c++
if (function_values.size() < 2 || function_values.size() > 3)
```

Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp]: Good point! Fixed.

**Comment by rmg-x** [comment]
Couldn't the size check just be changed to `size() == 2` ?

**Comment by trflynn89** [comment]
No, size of 3 is also valid.

**Pull request merged**